### ISSUE TITLE
Reorder Button disappears if entries count > pagination size setting ...

### ISSUE THREAD

**Issue by subpixelch** [statamic/v2-hub#1207]
### Expected behaviour
... Number based collections should be reorderable

### Actual behaviour
if entries count > pagination size setting there's no reorder button anymore ;)

### Steps to reproduce
1.  Have 11 entries in a collection
2.  Set Pagination Size to 10
3.  Try to reorder the entries

### Server configuration
Everywhere...

**Statamic version:**
2.5.1

**Updated from an older Statamic or fresh install:**
Ja

**Comment by jackmcdade** [comment]
That's on purpose. It's an unfortunate limitation of pagination.

**Comment by sheldonkotyk** [comment]
Just an idea, below the taxonomy area in the CP, if the order: numerical is set, have two dropdown with a list of numerically ordered titles where one could insert BEFORE or AFTER then choose the title.

**Comment by subpixelch** [comment]
@jackmcdade why is this closed? This is a feature that should work AFAIT... why not just show the whole list when `reorder` is pressed an turn the pagination off temporary?

**Comment by sheldonkotyk** [comment]
I agree that the functionality should work. Unfortunately, we can't show the whole list as after you hit a larger amount of entries the page will actually crash which is why we needed the pagination in the first place.

**Comment by jackmcdade** [comment]
You can increase the pagination size if you want. An easy workaround.

**Comment by dannyuk1982** [comment]
Could this link to a *very* lightweight page that show all entries with a drag/drop reorder and nothing else, regardless of pagination settings. Seems mad to hide it if pagination is used and it's very key functionality. Setting pagination to i.e. 1000 is a good workaround but then you lose the benefits of pagination for everything but reordering.

**Comment by sheldonkotyk** [comment]
after about 150 entries it was crashing in some browsers.

**Comment by gmcz** [comment]
I just want to plus-one the importance of having entries re-ordable by clients in the CP. I’m not tied to it happening a particular way, but it does need to be possible.

**An idea:**
Like Danny says above, a reorder-specific tool... but... perhaps as a modal when you click on “reorder”?

**Comment by wuhhh** [comment]
This is a major problem - if I have 50+ entries am I supposed to rename every single one every time one entry changes position? How is this even remotely usable for anyone? How do I change the pagination size for collections in the control panel and where is this in the docs?

**Comment by jasonvarga** [comment]
Settings > Control Panel > Pagination Size

or `pagination_size` in `site/settings/cp.yaml`

**Comment by jeremydouglas** [comment]
@gmcz did you ever pursue reordering entries further? Maybe an addon would be a temporary workaround until this improves in the Statamic core? I don't mind the modal idea as a temporary solution.

**Comment by jeremydouglas** [comment]
Being able to set a different `pagination_size` for assets and entries would be helpful.

**Comment by jeremydouglas** [comment]
fwiw, this is still my #1 feature request for Statamic :)

**Comment by wesort** [comment]
+1 for reordering large collections. I have a client with 70ish but this will increase to 100s. 

Along with drag-n-drop it would also be helpful to set the new order number manually in the CP. 

And as a temp fix, could the reorder button be greyed out (with a tooltip explanation)? And even if entries can't be moved between pages in the CP, would be nice if they could be reorderd within the current page.
